docs/meetups/03-12-2025/notes.md

```diff
@@ -9,15 +9,15 @@
 Speakers: Hongtao Yu (Meta), Yuanwei (Kevin) Fang (Meta), Manman Ren (Meta)
 
 Notes:
-* Pytorch 2.6 with Triton release branch 3.2 
+* Pytorch 2.6 with Triton release branch 3.2
```

Comment:
These changes are unrelated?

Reply:
I see https://github.com/triton-lang/triton/pull/7376/

Reply:
Agree. The formatting of these files is gone after I added upstream changes to my PR.